_meta/beat.yml

```diff
@@ -72,7 +72,21 @@ apm-server:
     # to all error and transaction documents sent to the frontend endpoint.
     #source_mapping:
 
-      # Source maps are are fetched from Elasticsearch and then kept in an in-memory cache for a certain time.
+      # Source maps are always fetched from Elasticsearch, by default using the output.elasticsearch configuration.
+      # A different instance must be configured when using any other output.
+      # This setting only affects sourcemap reads - the output determines where sourcemaps are written.
```

Comment:
I will phrase this more along the lines:
"if you use a logstash or kafka output, you should set this property to the elasticsearch host that logstash or kafka is writing to"...

Reply:
I'm worried that as we add outputs we'll forgot to make updates to this.